index.html

```diff
@@ -15,7 +15,7 @@
 
     <title>Utah DEQ Environmental Interactive Map</title>
   </head>
-  <body class="h-full w-full overflow-hidden">
+  <body class="h-full w-full overflow-hidden text-gray-700">
```

Comment:
**suggestion**: it's a best practice to put the text colors on the elements that use them. This makes it easier to add a dark mode. Just a suggestion, feel free to resolve without action.

Reply:
Interesting. Maybe this should be part of the utds plugin. I'm going to try to move forward for now.